sdcm/cluster_aws.py

```diff
@@ -675,6 +675,12 @@ def restart(self):
             try:
                 self.stop_scylla_server(verify_down=False)
 
+                # Moving var-lib-scylla.mount away, since scylla_create_devices fails if it already exists
+                mount_path = "/etc/systemd/system/var-lib-scylla.mount"
+                existance_check = self.remoter.run(f'sudo test -e {mount_path}', ignore_status=True)
```

Comment:
It's fine to directly remove it without checking if it exists or not.

`sudo rm -f /etc/systemd/system/var-lib-scylla.mount`

Reply:
@roy wanted me to save the file for debugging purposes